### PR TITLE
Story portfolio build complete

### PR DESCRIPTION
**Pull request by JKapella**
- Designed portfolio website (wireframe available here - https://docs.google.com/presentation/d/1UslObd5AaJhkfURtPAbaPRupwlq5gcKeBdOHiKviUY0/edit?usp=sharing)
- Design template built
- added about me section, portfolio (i real piece of work, 5 placeholders) and contact info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>James Kapella || Web Developer</title>
+    <link rel='stylesheet' href='normalize.css'>
```
Comment (charliecog) [inline index.html]: missing type=text/css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <link rel='stylesheet' href='styles.css'>
```
Comment (charliecog) [inline index.html]: missing type=text/css

File: index.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Oswald:200" rel="stylesheet">
```
Comment (charliecog) [inline index.html:5]: should include links to external fonts before the css file that calls them

File: index.html [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,300,400" rel="stylesheet">
```
Comment (charliecog) [inline index.html:6]: should include links to external fonts before the css file that calls them

File: index.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
```
Comment (charliecog) [inline index.html]: missing html lang attribute

File: index.html [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+<head>
+    <title>James Kapella || Web Developer</title>
+    <link rel='stylesheet' href='normalize.css'>
+    <link rel='stylesheet' href='styles.css'>
+    <link href="https://fonts.googleapis.com/css?family=Oswald:200" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200,300,400" rel="stylesheet">
+</head>
+<body>
+
```
Comment (charliecog) [inline index.html]: remove line break

File: index.html [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    <header>
+        <div class='container'>
```
Comment (charliecog) [inline index.html:12]: could the class of container be put on the header instead and the extra div removed?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+                        <a href='http://culturebosh.com' target='_blank'>
+                            <img src='img/gameIcon.png' alt='a games icon'> My Games
+                        </a>
+                    </li>
+                </ul>
+            </section>
+        </div>
+    </header>
+    <main>
+        <div class='container'>
```
Comment (charliecog) [inline index.html:48]: could the class of container be put on the main instead and the extra div removed?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+            <h2>PORTFOLIO</h2>
+            <h3>Scroll down</h3>
+            <img class='scrollingMouseImage pulsingImage' src='img/mousescrollitem.png' alt='Scroll-down icon'>
+            <a href='../w1_pilotShopRebuild/index.html' target='_blank'>
```
Comment (charliecog) [inline index.html]: this url is relative so will only work on your machine, just pop a # in until you have a live url to link to (same to any below as well)

File: index.html [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+                <article>
+                    <h3>PILOT SHOP</h3>
+                    <img src='img/pilotShopScreenshot.jpg' alt='A screenshot of my pilot shop rebuild'>
+                    <p>As an exercise in responsive web design we recreated the Pilot Shop digital storefront with HTML and CSS.</p>
+                </article>
+            </a>
+            <a href='' target='_blank'>
```
Comment (charliecog) [inline index.html]: best practice tip:
Fine this time so don't change, but hint for the future
a tags are inline elements and articles are block level elements
it is good practice not to put a block level element inside a default inline element. instead doing it the other way round, then changing the a tag to inline-block or block so you can assign a width and height

File: index.html [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+                <article>
+                    <h3>PLACEHOLDER PROJECT</h3>
+                    <img src='img/portfolioPlaceholder3.jpg' alt='More coming soon, placeholder image'>
+                    <p>This is a placeholder (lorem ipsum, etc). Expect to see more exciting projects coming soon to this page.</p>
+                </article>
+            </a>
+        </div>
+    </main>
+
```
Comment (charliecog) [inline index.html]: remove whitespace line

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,270 @@
...
+    margin-bottom: 0;
+}
+
+ul {
+    padding: 0;
+    text-align: center;
+}
+
+li {
+    display: inline-block;
```
Comment (charliecog) [inline styles.css:128]: tip: fine this time but try and get in the habit of using float, loads easier and doesn't come with built in bugs

**Review by charliecog — COMMENTED** [review]
Really good build, some impressive css and nice elegant html structure. A few bits here and there to pick up and a couple of tips that arent for implementing this time but useful best practice stuff to know for the future (all of which are splitting hairs so no major worry). Nice work!

File: index.html (hunk 5 above)
Comment (JKapella) [reply index.html]: Added

File: index.html (hunk 1 above)
Comment (JKapella) [reply index.html]: Added

File: index.html (hunk 2 above)
Comment (JKapella) [reply index.html]: Added

File: index.html (hunk 7 above)
Comment (JKapella) [reply index.html:12]: For this one, not easily, container div is a reactive element sitting inside the fixed header element, to make the container class work the same way on the header, i'd end up having to add several more lines of CSS to the header?

File: index.html (hunk 8 above)
Comment (JKapella) [reply index.html:48]: Again, not easily, as the 'main' element sets the relative position of the right half of the site, and the container adds some responsive margins for the content within that, if you reckon it's really ugly I can change it though...

File: index.html (hunk 9 above)
Comment (JKapella) [reply index.html]: Changed, thanks

File: index.html (hunk 10 above)
Comment (JKapella) [reply index.html]: Thanks, that makes a lot of sense, could probably have avoided using the article tags altogether and several lines of html.

File: index.html (hunk 11 above)
Comment (JKapella) [reply index.html]: done

File: styles.css (hunk 12 above)
Comment (JKapella) [reply styles.css:128]: Thanks, will do. Am still scared of float.

File: index.html (hunk 7 above)
Comment (JKapella) [reply index.html:12]: (reactive in that it uses margin 0 auto to allow the margins to move a bit with different sized screens)

File: index.html (hunk 8 above)
Comment (JKapella) [reply index.html:48]: Also, (sorry, not good at explaining) the container does the same job not matter what resolution/device view you're using - whereas the main and header sections behave differently depending on the device.

I don't think that was a very good justification.

**Review by charliecog — APPROVED** [review]
awesome job!

**Pull request merged**